### PR TITLE
Quote and weather

### PR DESCRIPTION
**Pull request by shubhamkumar648**
- add quotes and weather 
- fix CSS property of all component

### CODE REVIEW

**Review by RahulSingh9131 — APPROVED** [review]
mentioned feature seems to be working fine, also liked the idea of making custom hook for local storage. Just remove the commented code before merging.

**Review by aniketsingh13 — APPROVED** [review]
features are working fine.
 background wallpaper is not fullscreen. 
looking good to merge

File: src/App.css [hunk 1]
```diff
@@ -1,15 +1,81 @@
 
-
-
 .coverImage-mainContainer {
```
Comment (vaibhav-mahalle) [inline src/App.css:2]: you could have used the particular class for defining flex properties.

File: src/App.css [hunk 2]
```diff
@@ -1,15 +1,81 @@
...
+   z-index: -1;
+  }
+
+  .getTime {
+    justify-content: center;
+      display: flex;
+      padding-top: 5rem;
+    }
+
+  .focous-input {
```
Comment (vaibhav-mahalle) [inline src/App.css:43]: minor spelling mistake.

File: src/component/CoverImage.js [hunk 3]
```diff
@@ -20,17 +20,38 @@ export const CoverImage = () => {
   }, []);
 
   return (
-    <div className="coverImage-mainContainer flex">
-      <div className="currTime-container">
-        <Currtime />
+    <div className="coverImage-mainContainer">
```
Comment (vaibhav-mahalle) [inline src/component/CoverImage.js:23]: formatting could be a little better, you could use the prettier.

**Review by vaibhav-mahalle — COMMENTED** [review]
Added some suggestions, please look into them.

**Pull request merged**